gtest/src/system.rs

```diff
@@ -128,6 +128,18 @@ impl System {
         self.0.borrow_mut().store_new_code(&code)
     }
 
+    /// Saves code and associated meta to the storage and returns it's code hash.
+    pub fn submit_code_and_meta<P: AsRef<Path>>(&self, code_path: P, meta_path: P) -> CodeId {
```

Comment:
```suggestion
    pub fn submit_code_and_meta(&self, code_path: impl AsRef<Path>, meta_path: impl AsRef<Path>) -> CodeId {
```

To allow use different types for `code_path` and `meta_path`: current impl denies, for example,
`sys.submit_code_and_meta(String::from("../bin"), "../meta_bin")`